subprojects/core/src/main/java/org/gradle/initialization/StartParameterBuildOptions.java

```diff
@@ -649,4 +647,19 @@ public void applyTo(StartParameterInternal settings, Origin origin) {
             settings.setPropertyUpgradeReportEnabled(true);
         }
     }
+
+    public static class ProblemReportGenerationOption extends EnabledOnlyBooleanBuildOption<StartParameterInternal> {
+
+        public static final String LONG_OPTION = "disable-problem-report";
+        public static final String GRADLE_PROPERTY = "org.gradle.problem.report.disabled";
+
+        public ProblemReportGenerationOption() {
+            super(GRADLE_PROPERTY, CommandLineOptionConfiguration.create(LONG_OPTION, "(Experimental) disables HTML problem report generation"));
+        }
```

Comment:
🤔 Maybe the URL we show on the console should be alsow prefixed with `incubating:`